app/templates/admin/cceMinor.html

```diff
@@ -60,70 +60,146 @@ <h2>CCE Minor Progress</h2>
 <br> <br>
 <div class="row">
     <div class="col">
-        <h2>Interested Students</h2>
+        <h2>CCE Minor Candidates</h2>
     </div>
-    <div class="col-auto">
-        <a class="btn btn-success" role="button" href="#" data-bs-toggle="modal" data-bs-target="#addInterestedStudentsModal" style="margin:5px;">Add Students</a>
-        <button class="btn btn-primary" id="emailAllInterested">Email All</button>
+    <div class="col-auto d-flex align-items-center">
+      <a class="btn btn-success m-1" role="button" href="#" data-bs-toggle="modal" data-bs-target="#addInterestedStudentsModal">Add Students</a>
+      <div class="dropdown">
+        <button class="btn btn-primary dropdown-toggle" type="button" id="emailAllInterestedDropdown" data-bs-toggle="dropdown" aria-expanded="false">
+          Email All
+        </button>
+        <ul class="dropdown-menu" aria-labelledby="emailAllParticipantsDropdown">
+          <li><a class="dropdown-item" href="#" id="emailAllInterested">Interested</a></li>
+          <li><a class="dropdown-item" href="#" id="emailAllDeclared">Declared</a></li>
+          <li><hr class="dropdown-divider"></li>
+          <li><a class="dropdown-item" href="#" id="emailAll">Both Groups</a></li>
+        </ul>
+      </div>
     </div>
 </div>
-<table class="display" id="interestedStudentsTable">
-    <thead>
-        <tr>
-            <th>Name</th>
-            <th> Remove Student</th>
-        </tr>
-    </thead>
-    <tbody>
-        {% for student in interestedStudentsList %}
-            <tr>
-                <td style = "width: 82%">
-                    <a href="/profile/{{student.username}}/cceMinor" target="_blank">{{student.firstName}} {{student.lastName}}</a>
-                </td>
-                <td style="text-align:center">
-                  <button
-                    type="button"
-                    class="remove_interested_student btn btn-danger btn-sm"
-                    id = "{{student.username}}"
-                    aria-label="remove interested student">
-                    <span class="bi bi-trash h5 align-middle"></span>
-                  </button>
-                </td>
-            </tr>
-        {% endfor %}
-        <input id="interestedStudentEmails" hidden value="{{interestedStudentEmailString}}"/>
-    </tbody>
-</table>
+
+
+
+
+{% macro renderStudents(tabData, defaultActiveTab='interested') %}
```

Comment:
I think we have a folder with some other macros that you could potentially move this into to clean it up.